app/deserializers/forest_liana/resource_deserializer.rb

```diff
@@ -32,7 +32,15 @@ def extract_attributes
     def extract_attributes_serialize
       attributes_serialized.each do |attribute, serializer|
         value = @params[:data][:attributes][attribute]
-        @attributes[attribute] = JSON::parse(value)
+        if (!value)
+          @attributes[attribute] = nil
+        else
+          begin
+            @attributes[attribute] = JSON::parse(value)
+          rescue
+            raise "Bad format for attribute '#{attribute}'."
```

Comment:
You were supposed to create a specific error message to distinguish this error from unexpected ones